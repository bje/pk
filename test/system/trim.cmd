../../src/fieldx -t 1 2 3
