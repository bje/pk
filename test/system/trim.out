foo bar baz
arfle barfle gl:oop
